discord/ui/view.py

```diff
@@ -181,6 +184,7 @@ def __init__(self, *items: Item, timeout: Optional[float] = 180.0):
         self.__timeout_expiry: Optional[float] = None
         self.__timeout_task: Optional[asyncio.Task[None]] = None
         self.__stopped: asyncio.Future[bool] = loop.create_future()
+        self._message: Message | None = None
```

Comment:
Compatibility with older python versions
```suggestion
        self._message: Optional[Message] = None
```

Reply:
It even shows Optional stuff 2 lines above